common/main/zh.xml

```diff
@@ -13647,37 +13647,37 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<namePattern>{surname}{given-informal}</namePattern>
 		</personName>
 		<personName order="sorting" length="medium" usage="referring" formality="formal">
-			<namePattern>{surname-core}{given}{given2-initial}{surname-prefix}</namePattern>
+			<namePattern>{surname-core}{given}{given2}{surname-prefix}</namePattern>
 		</personName>
 		<personName order="sorting" length="medium" usage="referring" formality="informal">
 			<namePattern>{surname}{given-informal}</namePattern>
 		</personName>
 		<personName order="sorting" length="short" usage="referring" formality="formal">
-			<namePattern>{surname-core}{given-initial}{given2-initial}{surname-prefix}</namePattern>
+			<namePattern>{surname-core}{given}{given2}{surname-prefix}</namePattern>
 		</personName>
 		<personName order="sorting" length="short" usage="referring" formality="informal">
 			<namePattern>{surname}{given-informal}</namePattern>
 		</personName>
 		<sampleName item="givenOnly">
-			<nameField type="given">辛巴达</nameField>
+			<nameField type="given">宇瀚</nameField>
 		</sampleName>
 		<sampleName item="givenSurnameOnly">
-			<nameField type="given">三</nameField>
+			<nameField type="given">安怡</nameField>
 			<nameField type="surname">张</nameField>
 		</sampleName>
 		<sampleName item="given12Surname">
-			<nameField type="given">约翰</nameField>
-			<nameField type="given2">哈米什</nameField>
-			<nameField type="surname">华生</nameField>
+			<nameField type="given">俊年</nameField>
+			<nameField type="given2">杰思</nameField>
+			<nameField type="surname">陈</nameField>
 		</sampleName>
 		<sampleName item="full">
-			<nameField type="prefix">教授博士</nameField>
-			<nameField type="given">艾达·科妮莉亚</nameField>
-			<nameField type="given-informal">尼尔</nameField>
-			<nameField type="given2">伊娃·索菲亚</nameField>
-			<nameField type="surname-prefix">范·登</nameField>
-			<nameField type="surname-core">瓦尔夫</nameField>
-			<nameField type="surname2">贝克·施密特</nameField>
+			<nameField type="prefix">∅∅∅</nameField>
+			<nameField type="given">夏玉格</nameField>
+			<nameField type="given-informal">小宇</nameField>
+			<nameField type="given2">柏宇</nameField>
+			<nameField type="surname-prefix">∅∅∅</nameField>
+			<nameField type="surname-core">∅∅∅</nameField>
+			<nameField type="surname2"> ∅∅∅</nameField>
```

Comment:
Looks like there is a space in > ∅∅∅<

Reply:
Fixed, and search all source for [^>]∅∅∅[^<], finding none after.